superset-frontend/packages/superset-ui-chart-controls/src/sections/annotationsAndLayers.tsx

```diff
@@ -33,6 +34,7 @@ export const annotationsAndLayersControls: ControlPanelSectionConfig = {
           label: '',
           default: annotationLayers,
           description: t('Annotation Layers'),
+          renderTrigger: true,
```

Comment:
@stephenLYZ  the `renderTrigger == ture` means that client rerenders and doesn't send a query, but here is an exception. add a comment here. Thanks!